packages/adaptive_scaffold/example/test/adaptive_layout_demo_test.dart

```diff
@@ -2,11 +2,266 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:adaptive_scaffold/src/adaptive_layout.dart';
+import 'package:adaptive_scaffold/src/slot_layout.dart';
+import 'package:adaptive_scaffold/src/breakpoints.dart';
+import 'package:adaptive_scaffold/adaptive_scaffold.dart';
 import 'package:adaptive_scaffold_example/adaptive_layout_demo.dart' as example;
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 
+Future<MaterialApp> layout({
```

Comment:
You shouldn't need this method at all. Just using the updateScreen method should be sufficient. You shouldn't need to rebuild the entire app when you can just import the class you need and build it as is done in the updateScreen.